Gemfile

```diff
@@ -8,7 +8,10 @@ gem 'redis'
 gem 'secure_headers'
 gem 'sinatra'
 gem 'thin'
+gem 'daru'
 
 gem "rack-test", "~> 2.0", :group => :testing
 
 gem "test-unit", "~> 3.5", :group => :testing
+
+gem "matrix", "~> 0.4.2"
```

Comment:
Why is this needed?

Reply:
This is required as daru uses the matrix library which was removed from Ruby standard library after version 3.1
https://stackoverflow.com/questions/71991514/rails-require-cannot-load-such-file-matrix